### PR TITLE
Add tolerance and targetCurrency setting to Paymero

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccountConfig/Paymero.yaml [hunk 1]
```diff
@@ -15,3 +15,13 @@ allOf:
             format: password
         required:
           - apiKey
+      settings:
+        type: object
+        description: Paymero settings object.
+        properties:
+          tolerancePercentage:
+            type: string
+            description: Tolerance percentage for settled amount for crypto currency method.
```
Comment (colinbird) [inline openapi/components/schemas/GatewayAccountConfig/Paymero.yaml:24]: Maybe include an example?  Is it like "50" or ".5" or "50%" ?

Should it be an integer?

**Review by colinbird — DISMISSED** [review]

File: openapi/components/schemas/GatewayAccountConfig/Paymero.yaml [hunk 2]
```diff
@@ -15,3 +15,15 @@ allOf:
...
           - apiKey
+      settings:
+        type: object
+        description: Paymero settings object.
+        properties:
+          tolerancePercentage:
+            description: Tolerance percentage (from 1 to 100) for settled amount for crypto currency method.
+            type: integer
+            example: 5
+          targetCurrency:
```
Comment (ashkarpetin) [inline openapi/components/schemas/GatewayAccountConfig/Paymero.yaml:26]: Maybe use already existing CurrencyCode schema?

**Review by ashkarpetin — APPROVED** [review]

File: openapi/components/schemas/GatewayAccountConfig/Paymero.yaml [hunk 3]
```diff
@@ -15,3 +15,15 @@ allOf:
             format: password
         required:
           - apiKey
+      settings:
+        type: object
+        description: Paymero settings object.
+        properties:
+          tolerancePercentage:
+            description: Tolerance percentage (from 1 to 100) for settled amount for crypto currency method.
```
Comment (slavcodev) [inline openapi/components/schemas/GatewayAccountConfig/Paymero.yaml:23]: What about min and max?

File: openapi/components/schemas/GatewayAccountConfig/Paymero.yaml [hunk 4]
```diff
@@ -15,3 +15,15 @@ allOf:
...
+            type: integer
```
Comment (slavcodev) [inline openapi/components/schemas/GatewayAccountConfig/Paymero.yaml:24]:
```suggestion
            type: integer
            minimum: 1
            maximum: 100
```

Comment (slavcodev) [reply openapi/components/schemas/GatewayAccountConfig/Paymero.yaml:24]: What about min/max constraints?

**Review by slavcodev — APPROVED** [review]

**Pull request merged**